### PR TITLE
added dialysis files

### PR DESCRIPTION
**Pull request by camilafaccini**
Added dialysis template files and parameter sweep

### CODE REVIEW

**Review by jadhao — COMMENTED** [review]
The addition of dialysis files seems ok. Not reviewed in detail, at a high level.

File: model_extension/code_pcf/functions.cpp [hunk 1]
```diff
@@ -163,6 +163,33 @@ int select_gr(int vlpTypes, int *vlpOriginType, int *vlpTargetType, long double
...
       }
   }
+  else if (vlpTypes == 4)
+  {
+      cout << "\nfor a 4-component system, there are 4 self correlation functions (g11, g22, g33, g44) and 6 cross correlation function (g12, g13, g14, g23, g24, g34)" << endl;
+      cout << "which correlation function you want to compute: self or cross?" << endl;
+      cin >> grType;
+      if (grType == "self")
+      {
+          cout << "which self correlation you want to compute: between VLP type 1 and VLP type 1 (enter 1) OR VLP type 2 and VLP type 2 (enter 2) OR between VLP type 3 and VLP type 3 (enter 4 <-- THIS IS NOT A TYPO) OR between VLP type 5 and VLP type 5 (enter 5 <-- THIS IS NOT A TYPO)?" << endl;
```
Comment (jadhao) [inline model_extension/code_pcf/functions.cpp:173]: @camilafaccini VLP type 5 and VLP type 5 should be "VLP type 4 and VLP type 4" I think

File: model_extension/code_pcf/functions.cpp [hunk 2]
```diff
@@ -163,6 +163,33 @@ int select_gr(int vlpTypes, int *vlpOriginType, int *vlpTargetType, long double
...
+          cin >> selfType;
+          *vlpOriginType = selfType;
+          *vlpTargetType = *vlpOriginType;
+      }
+      else if (grType == "cross")
+      {
+          cout << "which cross correlation you want to compute: between VLP type 1 and VLP type 2 (enter 1 ENTER 2) OR VLP type 1 and VLP type 3 (enter 1 ENTER 4) OR VLP type 1 and VLP type 4 (enter 1 ENTER 5) OR between VLP type 2 and VLP type 3 (enter 2 ENTER 4) OR between VLP type 2 and VLP type 4 (enter 2 ENTER 5) OR between VLP type 2 and VLP type 4 (enter 2 ENTER 5) OR between VLP type 3 and VLP type 4 (enter 4 ENTER 5)?" << endl;
```
Comment (jadhao) [inline model_extension/code_pcf/functions.cpp:180]: VLP type 2 and VLP type 4 text repeated twice.

**Review by jadhao — COMMENTED** [review]
@camilafaccini left some comments, mostly minor typos in the text to the end user. surprised the code did not need much changes beyond what you have sent to incorporate 4 comp? 
Ideally, it will be good to fix the minor issues I have noted and update the PR, and then I approve. But if it works better for you for me to approve this PR and then you fix these things in your next PR, we can do that too.

File: model_extension/code_pcf/functions.cpp
Comment (camilafaccini) [inline model_extension/code_pcf/functions.cpp:1]: I've made the corrections indicated. Let me know if you find any mistakes or typos

**Comment by jadhao** [comment]
@camilafaccini approving.

**Pull request merged**